fixes.inc

```diff
@@ -15723,6 +15923,57 @@ native BAD_GetServerVarAsBool(const varname[]) = GetServerVarAsBool;
 	#define GetVehicleSeats__( FIXES_GetVehicleSeats(
 #endif
 
+/**
+ * <remarks>
+ * Checks if component is rear bullbar.
+ * </remarks>
+ */
+
+#if FIX_API
+	stock bool:FIXES_IsComponentRearBullbar(componentid)
+	{
+		if(componentid == 1109 || componentid == 1110)
+		{
+			return true;
+		}
+		return false;
+	}
+
+	#if _FIXES_SAMP
+		#define _ALS_IsComponentRearBullbar
+		#define IsComponentRearBullbar( FIXES_IsComponentRearBullbar(
+	#endif
+
+	#define _ALS_IsComponentRearBullbar__
+	#define IsComponentRearBullbar__( FIXES_IsComponentRearBullbar(
+#endif
+
+/**
+ * <remarks>
+ * Checks if component is front bullbar.
+ * </remarks>
+ */
+
+#if FIX_API
```

Comment:
This function existing reliles on `FIX_API` being `true`, but other parts of your code use it unconditionally, which means you'll get an error if `FIX_GetVehicleComponentInSlot` is `1` and `FIX_API` is `0`.

Reply:
I asked you before, should I make it static or implement all of API?

Reply:
I believe it is only used in one place in the code, so inline it there.  No need for a separate function.

Reply:
Done in https://github.com/pawn-lang/sa-mp-fixes/pull/149/commits/8f85355c1cb216367e9b7b38783f0f31068372d3